addon/components/single-event-objective-list.hbs

```diff
@@ -1,11 +1,24 @@
-<div class="single-event-objective-list">
-  <h2>
-    {{@title}}
+<div
+  class="single-event-objective-list"
+  data-test-single-event-objective-list
+  {{did-insert (set this.isExpanded @isExpandedByDefault)}}
```

Comment:
I like this, but wonder if the constructor is a better place to set defaults like this from arguments. Not sure. Just wondering.

Reply:
:heavy_check_mark: